pysim/compositesystem.pyx

```diff
@@ -23,24 +23,47 @@ cdef class CompositeSystem(SimulatableSystem):
         self.inputs = PysimVars._create(&_c_sys_local.inputs)
         self.outputs = PysimVars._create(&_c_sys_local.outputs)
         self.connections = Connections._create(&_c_sys_local.connectionHandler)
-        self.subsystems = {}
+        self.subsystems = OrderedDict()
```

Comment:
self.subsystems includes both ACTUAL subsystems and FAKE subsystems (adders :) ). It would be very good to be able to treat them separately. Any ideas?